src/gleam_stdlib.mjs

```diff
@@ -564,6 +564,8 @@ export function classify_dynamic(data) {
     return "Map";
   } else if (typeof data === "number") {
     return "Float";
+  } else if (data === null) {
+    return "null";
```

Comment:
My gut tells me `Return Nil`: `https://github.com/gleam-lang/stdlib/blob/6cb76cb53d728ce32c9c1ab7424788e9aab65ca4/src/gleam_stdlib.mjs#L20`

Reply:
I changed that. This was also the missing piece to add a test. Thanks 🙏🏾